osu.Game.Rulesets.Catch/UI/CatcherArea.cs

```diff
@@ -33,10 +33,10 @@ public CatcherArea(BeatmapDifficulty difficulty = null)
         {
             RelativeSizeAxes = Axes.X;
             Height = CATCHER_SIZE;
-            Child = MovableCatcher = new Catcher(difficulty)
-            {
-                AdditiveTarget = this,
-            };
+            Child = MovableCatcher = new Catcher(difficulty);
+
+            // this property adds containers to 'this' so it must not be set in the object initializer.
+            MovableCatcher.AdditiveTarget = this;
```

Comment:
If it's unsafe to call in the object initialiser it shouldn't be a property at all...? I'd say this needs to be rectified, as it looks like a foot-gun.